Jenkinsfile.properties

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env groovy
+
+// Loads the Jenkins pipeline properties configuration
+def pipelineParameters = parameters(
+  [
+    booleanParam(name: 'DRY', defaultValue: true, description: 'test run'),
+    stringParam(name: 'REGIONS', defaultValue: 'us-west-2,us-east-1', description: 'comma separated list of aws regions'),
+    stringParam(name: 'AWS_ACCOUNT_CREDENTIAL_IDS', defaultValue: '', description: 'Comma separated Jenkins credential IDs for AWS accounts. Acceptable values are: development-aws,integration-aws,staging-aws,production-aws,engops-aws')
```

Comment:
mention that if its empty, then defaults will be used depending on if its a merge or a pull request